packages/react/src/components/IconButton/index.tsx

```diff
@@ -36,7 +36,7 @@ const IconButton = forwardRef<ClickableElement, IconButtonProps>(
         $variant={variant}
         $isActive={isActive}
       >
-        <pixiv-icon name={icon} />
+        <pixiv-icon aria-hidden="true" name={icon} />
```

Comment:
This changes all IconButton. If it's too board to decide I'll make this a separate PR.